examples/js/loaders/FBXLoader.js

```diff
@@ -3986,43 +3986,51 @@ THREE.FBXLoader = ( function () {
 
 		// Global Shear*Scaling
 		var lParentTM = new THREE.Matrix4();
-		var lLSM;
-		var lParentGSM;
-		var lParentGRSM;
+		var lLSM = new Matrix4();
+		var lParentGSM = new Matrix4();
+		var lParentGRSM = new Matrix4();
+		
+		var lParentTM_inv = new Matrix4().getInverse( lParentTM );
```

Comment:
Just `new Matrix4()` is probably OK here since `getInverse` is called on the identity matrix.

Reply:
Oh, I've just noticed this comment. Well, we can do further clean up PRs 🙂